<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node pkg="control_hierarchy" type="dispatcher.py" name="dispatcher">
	<param name="control_signal_topic"                  value="/lli/ctrl_request"/>
	<param name="car_pose_topic"                        value="/SVEA2/pose"/>
	<param name="path_topic"                            value="/SVEA2/path"/>
	<param name="waypoint_topic"                        value="/leader/pose"/>
	<param name="command_controller_topic"              value="/start_stop_controller"/>
	<param name="/command_controller_follow_topic"      value="/start_stop_follow"/>
    <param name="obstacles_topic"                       value="/obstacles"/>
	</node>
</launch>


